api/envoy/extensions/filters/http/ext_proc/v3/ext_proc.proto

```diff
@@ -275,6 +275,20 @@ message ExternalProcessor {
   // backend stream lifetime. In this case, Envoy will eventually timeout the external processor stream according to this time limit.
   // The default value is 5000 milliseconds (5 seconds) if not specified.
   google.protobuf.Duration deferred_close_timeout = 19;
+
+  // [#not-implemented-hide:]
+  // Send body to the side stream server once it arrives without waiting for the header response from that server.
+  // It only works for STREAMED body processing mode. For any other body processing modes, it is ignored.
+  //
+  // The server has two options upon receiving a header request:
+  // 1. Instant Response: Send the header response as soon as the header request is received.
+  // 2. Delayed Response: Wait for the body before sending any response.
+  // If the server chooses the second option, it has two further choices:
+  // 3. Separate Responses: Send the header response first, followed by separate body responses.
```

Comment:
We probably can improve the formatting (like using different indentation or  // *)
(3) and (4) do not follow (1) and (2) above, and technically they are not at the same bullet point level.

It can be done in next PR when #not-implemented-hide is removed.